example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/usr/local/Caskroom/flutter/1.2.1/flutter"
-export "FLUTTER_APPLICATION_PATH=/Users/rodydavis/Developer/GitHub/plugins/packages/flutter_whatsnew/example"
+export "FLUTTER_ROOT=C:\Users\rebar\fvm\versions\stable"
```

Comment:
I believe this whole file shouldn't be committed and it is better to remove it as the comment on line 2 suggests.

Reply:
yup will revert that